assets/js/app.js

```diff
@@ -0,0 +1,59 @@
+(function() {
+var weapon1 = $('.weapon1');
+var weapon2 = $('.weapon2');
+var special = $('.special');
+
+var heroes = [Mal];
+var enemies = [allianceSoldier];
+function Game() {
+  this.on(takeTurnWithWeapon, function() {
```

Comment:
The on function accepts a string as its first argument: http://backbonejs.org/#Events

Here you will receive an error `takeTurnWithWeapon` is undefined.